playbooks/provisioner/openstack/main.yaml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
       - { role: openstack/nova, action: setup }
+
+- include: plugins.yml
```

Comment:
It's suppose to gather all tasks that are extra steps for openstack provisioner. It's not ovb specific.